documentation/features/cpuset.md

```diff
@@ -0,0 +1,55 @@
+# Cpuset cgroup.
+
+From Kernel documentation:
+
+_" Cpusets provide a mechanism for assigning a set of CPUs and Memory Nodes to
+a set of tasks."_
+
+The Kata agent brings compatibility the cgroup cpuset CPU on the guest side.
+
+The cpuset CPU cgroup will be applied on two events:
+
+- containers creation.
```

Comment:
Suggest removing the periods after the bulleted items because they are not complete sentences